lib/asyncapiSchemaFormatParser.js

```diff
@@ -0,0 +1,46 @@
+const Ajv = require('ajv');
+const ParserError = require('./errors/parser-error');
+const asyncapi = require('@asyncapi/specs');
+const { improveAjvErrors } = require('./utils');
+
+module.exports = {
+  parse,
+  getMimeTypes
+};
+
+async function parse({ message, oryginalAsyncAPIDocument, fileFormat, parsedAsyncAPIDocument }) {
+  const ajv = new Ajv({
+    jsonPointers: true,
+    allErrors: true,
+    schemaId: 'id',
+    logger: false,
+  });
+  const payloadSchema = preparePayloadSchema(asyncapi[parsedAsyncAPIDocument.asyncapi]);
+  const validate = ajv.compile(payloadSchema);
+  const valid = validate(message.payload);
+
+  if (!valid) throw new ParserError({
+    type: 'validation-errors',
+    title: 'There were errors validating the message payload against the AsyncAPI schema format.',
```

Comment:
What about:

```suggestion
    title: 'This is not a valid AsyncAPI Schema Object.',
```

This way we use the same vocabulary as the spec ([Schema Object](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#schemaObject)) and make it clear these are user errors and not other kinds of errors like the parsed had a problem parsing it. In short, it's user's fault and they should fix it.